testdata/config/scylla.yaml

```diff
@@ -8,5 +8,4 @@ client_encryption_options:
   keyfile: /etc/scylla/db.key
   truststore: /etc/scylla/ca.crt
   require_client_auth: true
-# when using 5.4.x we have to specify force_schema_commit_log option
-force_schema_commit_log: true
```

Comment:
Have you deleted it intentionaly, since it is used by [dirver-matrix](https://github.com/scylladb/gocql-driver-matrix/blob/master/run.py) I do not recommend deleting it

Reply:
I think it was only necessary to specify it in 5.4.x. In the 6.0 it should be enabled be default